package.json

```diff
@@ -73,15 +73,16 @@
     "okapiInterfaces": {
       "acquisitions-units": "1.1",
       "batch-group": "1.0",
+      "batch-voucher.export-configurations": "1.0",
       "configuration": "2.0",
       "finance.funds": "1.0",
-      "invoice": "3.0 3.1",
+      "invoice": "3.3",
       "order-lines": "1.0",
       "organizations-storage.organizations": "3.0",
       "tags": "1.0",
-      "users": "15.0 15.1",
+      "users": "15.0",
```

Comment:
it's very questionable. What is the reason?
 I see 2 options here:

1. to leave as is 
2. to leave 15.1

Reply:
Left `15.1`